apps/web/src/components/file-browser/FileList.tsx

```
@@ -1,31 +1,35 @@
   onContextMenu: (event: MouseEvent, item: FolderChild) => void;
   /** Callback when drag starts */
   onDragStart: (event: DragEvent, item: FolderChild) => void;
-  /** Callback when an item is dropped onto a folder */
+  /** Callback when items are dropped onto a folder */
   onDropOnFolder?: (
-    sourceId: string,
-    sourceType: 'file' | 'folder',
+    items: DragItem[],
     sourceParentId: string,
     destFolderId: string
   ) => void;
```

ESLint

[error] Replace `⏎····items:·DragItem[],⏎····sourceParentId:·string,⏎····destFolderId:·string⏎··` with `items:·DragItem[],·sourceParentId:·string,·destFolderId:·string`

(prettier/prettier)